### DISCUSSION TITLE
problem with video playback

### DISCUSSION THREAD

**Discussion by ceewanna** [roflcoopter/viseron#645] [General]
I have 2 systems running viseron. One is manjaro/arch wiith xfce4 and the other is manjaro/arch with gnome. The latter can play back recordings while the former shows black screen  but still has audio. I could play the vdo files (mp4) via mpv wiithout problems. What are the settings/packages required for web playback or where should I look to identify problems?

**Comment by roflcoopter** [comment]
Good question, one that i cant answer unfortunately.
One thing that comes to mind is the encoding of your cameras, are they `h264` or `h265`?

**Reply by ceewanna** [reply]
Most of my cameras are h265. There are a few h264. I haven't tested all of them but I remember testing only those h265.

**Reply by roflcoopter** [reply]
I am thinking that could be the case, h265 may not be supported in all OSes/browsers.
You can re encode the videos to h264 by setting `codec: h264` under `recorder`

**Reply by ceewanna** [reply]
Two h264 cameras can play on xfce. The rest are  2 brands of h265 cameras. One brand can play on gnome but not xfce while the other can't play on both desktops. This one has some sort of h265+ feature but I am quite sure I opted out of this feature.

I might have to live with no being able to play back on web otherwise I will face storage problem if I convert everything to h264..

**Reply by ceewanna** [reply]
From my testing, the webpage can't play hevc video format but h264 is fine. MPV can play both hevc and h264.
